stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -251,6 +260,20 @@ public void onChildRemoved(Object parentElement, Object childElement) {
 
       removeElementTree(childElement);
     }
+
+    @Override
+    public void onInspectRequested(Object element) {
```

Comment:
It's named this way so as to match the name of DOM.inspectNodeRequested, which is what this call will directly map to (and is intended to be thought of in that way). Except it isn't a node, so we omit that word. And the other methods omit 'element', so we don't replace 'node' with 'element.'